extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -165,6 +168,58 @@ async function handleDeleteObjectCommand(context: azdata.ObjectExplorerContext,
 	});
 }
 
+async function handleRenameObjectCommand(context: azdata.ObjectExplorerContext, service: IObjectManagementService): Promise<void> {
+	const connectionUri = await getConnectionUri(context);
+	if (!connectionUri) {
+		return;
+	}
+	const nodeTypeDisplayName = getNodeTypeDisplayName(context.nodeInfo.nodeType);
+	const originalName = context.nodeInfo.metadata.name;
+	const newName = await vscode.window.showInputBox({
+		title: localizedConstants.RenameObjectDialogTitle,
+		value: originalName,
+		validateInput: (value: string): string | undefined => {
+			if (!value) {
+				return localizedConstants.NameCannotBeEmptyError;
+			} else {
+				// valid
+				return undefined;
+			}
+		}
+	});
+
+	// return if no change was made or the dialog was canceled.
+	if (newName === context.nodeInfo.label || !newName) {
```

Comment:
Is `context.nodeInfo.label` the same as `originalName`?

Reply:
good catch. I'll fix it.